src/TorchSharp/NN/Bilinear.cs

```diff
@@ -13,46 +13,57 @@ namespace Modules
     {
         public sealed class Bilinear : Module<Tensor, Tensor, Tensor>
         {
-            internal Bilinear(IntPtr handle, IntPtr boxedHandle) : base(handle, boxedHandle) { }
-
-            public new static Bilinear Load(string modelPath)
+            internal Bilinear(long in1_features, long in2_features, long out_features, bool hasBias = true, Device? device = null, ScalarType? dtype = null) : base(nameof(Bilinear))
             {
-                var res = Module<Tensor, Tensor>.Load(modelPath);
-                return new Bilinear(res.handle.DangerousGetHandle(), IntPtr.Zero);
+                weight = torch.empty(out_features, in1_features, in2_features, device: device, dtype: dtype).AsParameter();
+                var bound = 1 / Math.Sqrt(weight!.shape[1]);
+
+                init.uniform_(_weight, -bound, bound);
+
+                if (hasBias) {
+                    bias = torch.empty(out_features, device: device, dtype: dtype).AsParameter();
+                    var (fanIn, _) = init.CalculateFanInAndFanOut(weight);
+                    init.uniform_(_bias, -bound, bound);
+                }
+                //NOTE: it's important not to call 'RegisterComponents' here.
```

Comment:
Adding why would be helpful for future readers. I assume it is called in `base`?